governance/policies/archiving.md

```diff
@@ -0,0 +1,107 @@
+```
+status: propsal
+last-updated: 2024-11-28
+```
```

Comment:
this metadata is no longer needed, I guess... the status is soon in action :) and the last-updated is now provided by git.